debugger/src/main/kotlin/org/rust/debugger/RsDebuggerToolchainService.kt

```diff
@@ -40,10 +40,7 @@ class RsDebuggerToolchainService {
         val (frameworkPath, frontendPath) = when {
             SystemInfo.isMac -> "LLDB.framework" to "LLDBFrontend"
             SystemInfo.isUnix -> "lib/liblldb.so" to "bin/LLDBFrontend"
-            SystemInfo.isWindows -> {
-                val binaryDir = "${if (SystemInfo.is32Bit) "x86" else "x64"}/bin"
-                "$binaryDir/liblldb.dll" to "$binaryDir/LLDBFrontend.exe"
-            }
+            SystemInfo.isWindows -> "bin/liblldb.dll" to "bin/LLDBFrontend.exe"
```

Comment:
I've noticed that we download LLDB x64 in case of 64-bit Windows and LLDB x86 in case of 32-bit Windows (see `lldbUrls`). Just wondering, what happens if someone uses
1. `stable-i686-pc-windows-msvc` Rust on 64-bit Windows
2. `stable-x86_64-pc-windows-msvc` Rust on 32-bit Windows

I've tried the first case and it works fine

Reply:
I think it's the same situation when someone tries to compile code to non-host platform.
Currently, we do nothing with it on any platform.

> I've tried the first case and it works fine

In general, a user can be not able to run the final binary but in the case of Windows, I suppose it works because 64bit Windows can run 32bit binaries. 
Looks like we should notify users when target platform of binary is not matched with the host target on any platform and in any IDE (including CLion). For example, there is `i686-unknown-linux-gnu` target for Linux.
But I don't want to do it in this PR. It should be part of cross-compilation support in the plugin